tests/async/test_browsercontext.py

```diff
@@ -429,7 +429,7 @@ async def test_route_should_intercept(context, server):
     def handle(route, request):
         intercepted.append(True)
         assert "empty.html" in request.url
-        assert request.headers["user-agent"]
+        assert request.headers["User-Agent"]
```

Comment:
We have to do the lowercasing when converting header to Dict. Something like this: https://github.com/microsoft/playwright/blob/master/src/utils/utils.ts#L121-L126